functests/1_performance/performance.go

```diff
@@ -781,6 +782,19 @@ func validatTunedActiveProfile(nodes []corev1.Node) {
 	var err error
 	var out []byte
 	activeProfileName := components.GetComponentName(testutils.PerformanceProfileName, components.ProfileNamePerformance)
+
+	// check if some another Tuned profile overwrites PAO profile
+	tunedList := &tunedv1.TunedList{}
+	if err := testclient.Client.List(context.TODO(), tunedList); err != nil {
+		println(err)
```

Comment:
What does `println` is doing and where does it define?

Reply:
and why the test line passed if this is undefined?

Reply:
TIL: https://golang.org/pkg/builtin/#println https://play.golang.org/p/8ltQKd03z9X

Reply:
hm `Println is useful for bootstrapping and debugging; it is not guaranteed to stay in the language.`, let's just use klog it anyway already used under the method

Reply:
agreed :) at least now we knw why it compiled fine :)

Reply:
to be honest, now I can't find from a which corner of my head I've taken this... 

changed to `klog.Error(err)`